spec/features/articles_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe "Managing articles" do
+  describe "viewing the index page" do
+    let!(:post_1) { create :article }
+    let!(:post_2) { create :article }
+
+    it "shows all articles" do
+      visit '/articles'
+      expect(page).to have_content(post_1.title)
+      expect(page).to have_content(post_2.title)
+    end
```

Comment:
Any concern for testing the ordering of articles you have set up in the controller?